decimal_test.go

```diff
@@ -2279,14 +2279,64 @@ func TestDecimal_Cmp2(t *testing.T) {
 }
 
 func TestPow(t *testing.T) {
-	a := New(4, 0)
-	b := New(2, 0)
+	a := NewFromFloat(1.15)
```

Comment:
For tests, we are using the "test table" approach,. Could you do the same thing, just to stick to a single convention?